core/contracts/financial-templates/implementation/PricelessPositionManager.sol

```diff
@@ -379,6 +384,17 @@ contract PricelessPositionManager is FeePayer, AdministrateeInterface {
         emit SettleExpiredPosition(msg.sender, totalRedeemableCollateral.rawValue, tokensToRedeem.rawValue);
     }
 
+    /**
+     * @notice If token holders or sponsors do not redeem their positions for underlying collateral within
+     * a pre-defined `siphonDelay`, all underlying collateral is sent to the Store. This delay is set to
+     * be sufficiently long such that token holders have a reasonable period of time to withdrawal (>6 months).
```

Comment:
Small nit: we don't technically control how large this value is -- the deployer does.
```suggestion
     * a pre-defined `siphonDelay`, all underlying collateral is sent to the Store. This delay should be set to
     * be sufficiently long such that token holders have a reasonable period of time to withdrawal (>6 months).
```